Dockerfile

```diff
@@ -11,12 +11,15 @@ RUN apt-get update && \
     wget -qO- https://storage.googleapis.com/download.dartlang.org/linux/debian/dart_stable.list > /etc/apt/sources.list.d/dart_stable.list && \
     apt-get update && \
     apt-get install --no-install-recommends -y \
+        git \
```

Comment:
These new packages a required to successfully install statsmodels and lighting Python packages. The image was broken all this time.

Reply:
Ooops, I'm very sorry that I haven't done the needed updates in corresponding PRs!